src/pug/components/input/input.pug

```diff
@@ -0,0 +1,11 @@
+mixin input(data)
+  .input-item 
+    span= data
+      |:
+      b *
```

Comment:
Thẻ này là thẻ label chứ nhỉ. Ngoài ra thì cái dấu * required em thêm bằng :after :before nhé.
Thông thường thẻ label có thuộc tính `for` đi kèm với `id` của input để input có thể focus khi nhấn vào label.
Tuy nhiên, em có thể lược bỏ 2 thuộc tính for với id bằng cách
```
<label>
    {label}
    <input type='text' />
</label>
```